content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -29,8 +29,7 @@ If the load decreases, and the number of Pods is above the configured minimum,
 the HorizontalPodAutoscaler instructs the workload resource (the Deployment, StatefulSet,
 or other similar resource) to scale back down.
 
-Horizontal pod autoscaling does not apply to objects that can't be scaled (for example:
-a {{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}}.)
+Horizontal pod autoscaling does not apply to objects that can't be scaled (e.g. a {{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}}).
```

Comment:
We recommend that you keep the lines wrapped and not use 'e.g.' to replace 'for example'.
Thanks

Reply:
Refer to the style guide: https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases